lib/MusicBrainz/Server/Controller/Edit.pm

```diff
@@ -259,7 +259,7 @@ sub subscribed : Local RequireAuth {
     my ($self, $c) = @_;
 
     my $only_open = 0;
-    if ($c->req->query_params->{open} eq '1') {
+    if (($c->req->query_params->{open} // '') eq '1') {
```

Comment:
Why use the empty string comparison here instead of using defined like elsewhere?

Reply:
Because it is more concise, and [logicial defined-or operator](https://perldoc.perl.org/perlop.html#Logical-Defined-Or) does test if defined too. Amended other changes to use it if appropriate with d102a748e1, 6fe24002ca,  34a1b6b503, and 17fc7d607a.